### PR TITLE
fix(FEV-1222): The clickable area around the markers should be expended to easily reach a specific question

### PR DESCRIPTION
**Pull request by semarche-kaltura**
- clickable area around the markers expended;
- a11y;
- fix should cover issue https://kaltura.atlassian.net/browse/FEV-1221

### CODE REVIEW

File: src/components/ivq-bottom-bar/ivq-bottom-bar.tsx [hunk 1]
```diff
@@ -1,33 +1,51 @@
...
 
 interface IvqBottomBarProps {
   onPrev?: () => void;
   onNext?: () => void;
   questionCounter: string | VNode<{}>;
 }
 
-export const IvqBottomBar = ({onPrev, onNext, questionCounter}: IvqBottomBarProps) => {
+const translates: QuizTranslates = {
+  nextQuestionButtonAriaLabel: <Text id="ivq.next_question_button_area_label">Next Question button</Text>,
```
Comment (RoyBregman) [inline src/components/ivq-bottom-bar/ivq-bottom-bar.tsx:17]: No need to write button in the area label. The screen reader knows it is a button if the tag is button or it has a role of a button

File: src/ivq.tsx [hunk 2]
```diff
@@ -76,10 +77,10 @@ export class Ivq extends KalturaPlayer.core.BasePlugin {
       const questionBunchMap = new Map<string, Array<number>>();
       let questionBunch: Array<number> = [];
       qqm.forEach((qq: QuizQuestion) => {
-        questionBunch.push(qq.index);
-        if (qq.startTime === qq.next?.startTime) {
+        if (qq.startTime === qq.prev?.startTime) {
           return;
         }
+        questionBunch.push(qq.index);
```
Comment (RoyBregman) [inline src/ivq.tsx:83]: This another bug fix you added?

File: src/ivq.tsx [hunk 3]
```diff
@@ -142,6 +154,18 @@ export class Ivq extends KalturaPlayer.core.BasePlugin {
     }
   };
 
+  private _serOverlay = (fn: Function) => {
```
Comment (RoyBregman) [inline src/ivq.tsx:157]: Not sure I understand what this does

File: src/questions-visual-manager.tsx [hunk 4]
```diff
@@ -19,17 +19,17 @@ const {EventType} = core;
...
+  public removeQuestionOverlay: Function | null = null;
   constructor(
     private _getQuizQuestionMap: () => QuizQuestionMap,
     private _player: KalturaPlayerTypes.Player,
     private _eventManager: KalturaPlayerTypes.EventManager
   ) {}
 
   public onQuestionCuepointActive({id}: KalturaQuizQuestion) {
-    if (this.quizQuestionJumping) {
+    if (this.quizQuestionJumping || this._player.paused) {
```
Comment (RoyBregman) [inline src/questions-visual-manager.tsx:32]: another bug fix?

**Review by RoyBregman — CHANGES_REQUESTED** [review]

File: src/ivq.tsx (hunk 2 above)
Comment (semarche-kaltura) [reply src/ivq.tsx:83]: no, fix for markers and their area labels.
Previous implementation - if we have 3 markers in the same time - latest one displayed.
New implementation - if we have 3 markers in the same time - first one displayed.
We need the changes for correct area label on marker.

File: src/ivq.tsx (hunk 3 above)
Comment (semarche-kaltura) [reply src/ivq.tsx:157]: typo, fixed.
Method set IVQ Overlay that allow us remove the overlay component once we need it.

File: src/questions-visual-manager.tsx (hunk 4 above)
Comment (semarche-kaltura) [reply src/questions-visual-manager.tsx:32]: no, changes related to a11y for markers

**Review by RoyBregman — APPROVED** [review]

**Pull request merged**